packages/apollo-codegen-scala/src/language.ts

```diff
@@ -120,12 +119,12 @@ export function methodDeclaration(
     .join(", ");
 
   generator.printOnNewline(
-    `def ${methodName}(${paramsSection})` + (closure ? " = " : "")
+    `def ${methodName}(${paramsSection})` + (closure ? " =" : "")
```

Comment:
Just double checking that the space removal was intentional, since it looks like it could be accidental 🤷‍♂️

Reply:
Yeah, this is intentional. While reviewing snapshots I noticed that there was an extra space between the `=` and the following `{` so this fixes it.